library/src/main/res/values/attrs.xml

```diff
@@ -2,6 +2,7 @@
 <resources>
     <declare-styleable name="LinkBuilder">
         <attr name="defaultLinkColor" format="color" />
+        <attr name="defaultLinkHighlightColor" format="color" />
```

Comment:
Last one, lets change this to something like `defaultTextColorOfHightlightedLink`